doc/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  doc:
+    image: kuzzleio/documentation:dev
```

Comment:
Where is the docker file for this image?

Reply:
[here](https://github.com/kuzzleio/documentation/blob/3-dev/Dockerfile)